nucypher/blockchain/eth/registry.py

```diff
@@ -351,7 +356,7 @@ def __init__(self, filepath: str = None, *args, **kwargs):
 
     def search(self, beneficiary_address: str = None, contract_address: str = None):
         if not (bool(beneficiary_address) ^ bool(contract_address)):
-            raise ValueError("Pass contract_owner or contract_address, not both.")
+            raise ValueError("Pass either beneficiary_address or contract_address.")
```

Comment:
Can you include what input was received in the message here?

Reply:
✅